### ISSUE TITLE
Recurring Events pretty Please

### ISSUE THREAD

**Issue by zadadegdh** [Davepar/gcalendarsync#69]
So would love the addon to do reoccuring events.  Esp for scheduled year long meeting set up.  If a thing happens only 2-3 times a year I can just enter those individually. Thanks

**Comment by pl9999** [comment]
Great utility:) I also need this recurring event functionality.
My use case is to be able sync a series of recurring events from a calendar to a Google sheet, and later sync changes from one or all of those recurring calendar events to the Google sheet (syncing recurring events from the Google sheet to the calendar would be bonus, but it's not my use case for now).
Thanks!

**Comment by mzrenegade68** [comment]
Yes, I agree. The recurring event functionality would be a great addition.

Thanks
